integration_tests/integration/eventHandlers.spec.ts

```diff
@@ -50,11 +62,18 @@ context('Event handlers', () => {
     it('should listen to the offender manager changed event and call endpoint to update responsible COM', () => {
       cy.task('stubGetAnOffendersManagers')
       cy.task('stubGetStaffDetailsByStaffId')
+      cy.task('stubUpdateResponsibleCom')
```

Comment:
This was not already stubbed. The test did not fail for some reason.. but it threw a warning